stixcore/tmtc/tm/tm_21.py

```diff
@@ -36,11 +36,20 @@ def group_repeaters(self):
             subpacket.NIX00260 = param
 
     def get_decompression_parameter(self):
-        # https://github.com/i4Ds/STIX-FSW/issues/953
+
         params = super().get_decompression_parameter()
-        if self.data_header.datetime:
-            # TODO do some special treatment
-            return params
+
+        # https://github.com/i4Ds/STIX-FSW/issues/953
+        # In older version of the FSW the trigger were incorrectly compressed with the count
+        # compression scheme so need to modify default parameters
+        idb = self.idb.get_idb(obt=self.data_header.datetime)
+        idb_version = tuple(map(int, idb.version.split('.')))
+        if idb_version < (2, 26, 33):
```

Comment:
cool that this is done now

Reply:
Do you know what version of the FSW this was fixed in this was just a guess?